### PR TITLE
start adding effects

### PR DESCRIPTION
**Pull request by JBaczuk**
Adds a pulse effect for testing when you click play in the frontend on a device

### CODE REVIEW

File: api/colors.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
```
Comment (JBaczuk) [inline api/colors.json:1]: @YeonV I think we have this already in https://github.com/LedFx/ledfx_rewrite/blob/main/color/predefined.go

**Comment by shauneccles** [comment]
Are we ready to merge this?

**Comment by JBaczuk** [comment]
> Are we ready to merge this?

We can, but we should probably remove some redundant code, I'll do that.

**Comment by carterpeel** [comment]
> > Are we ready to merge this?
> 
> We can, but we should probably remove some redundant code, I'll do that.

Are you able to fix the conflicts so we can merge this?

**Comment by JBaczuk** [comment]
> > > Are we ready to merge this?
> > 
> > 
> > We can, but we should probably remove some redundant code, I'll do that.
> 
> Are you able to fix the conflicts so we can merge this?

Hi, sorry I've been busy I will try to take a look soon

**Comment by JBaczuk** [comment]
@carterpeel Just went through and fixed all the conflicts. It looks like the effects module was removed, not sure why. So I removed all references to it.

**Pull request merged**